subsys/spm/Kconfig

```diff
@@ -135,3 +138,9 @@ config SPM_NRF_SAADC_NS
 endif # IS_SPM
 
 endmenu
+
+if SPM || IS_SPM
```

Comment:
perhaps @ulfalizer can comment on that; where is the best place to set a new default name for this library?

Reply:
I wanted to have this here, since it assures both that the name is unique to the SPM, and that it is set correctly for both the S and NS firmware.